conf/base.config

```diff
@@ -51,22 +51,22 @@ process {
   }
   withName: BWA_INDEX {
     cpus   = { check_max( 1     * task.attempt, 'cpus'   ) }
-    memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+    memory = { check_max( 1.GB * task.attempt, 'memory'  ) }
     time   = { check_max( 4.h   * task.attempt, 'time'   ) }
   }
-  withName: BAM_SORT_SAMTOOLS:SAMTOOLS_SORT{
+  withName: SAMTOOLS_SORT {
     cpus   = { check_max( 1     * task.attempt, 'cpus'   ) }
-    memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+    memory = { check_max( 1.GB * task.attempt, 'memory'  ) }
     time   = { check_max( 1.h   * task.attempt, 'time'   ) }
   }
-  withName: BAM_SORT_SAMTOOLS:SAMTOOLS_INDEX{
+  withName: SAMTOOLS_INDEX {
     cpus   = { check_max( 1     * task.attempt, 'cpus'   ) }
-    memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+    memory = { check_max( 1.GB * task.attempt, 'memory'  ) }
     time   = { check_max( 1.h   * task.attempt, 'time'   ) }
   }
-  withName: BAM_SORT_SAMTOOLS:SAMTOOLS_SORT{
+  withName: SAMTOOLS_STATS {
     cpus   = { check_max( 1     * task.attempt, 'cpus'   ) }
-    memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+    memory = { check_max( 1.GB * task.attempt, 'memory'  ) }
     time   = { check_max( 1.h   * task.attempt, 'time'   ) }
```

Comment:
As these three resource requirements are exactly the same as the `process_low` requirements, they can be removed, and it is only necessary to specify the `label 'process_low'` inside those processes.

Reply:
Resolved apart from ` withName: SAMTOOLS_SORT` which needed overriding from `process_medium` for bacterial files